src/core/public/notifications/toasts/toasts_api.tsx

```diff
@@ -87,10 +87,6 @@ export interface ErrorToastOptions extends ToastOptions {
    * message will still be shown in the detailed error modal.
    */
   toastMessage?: string;
-  /**
-   * Unique ID for the toast. Can be used to prevent duplicate toasts on re-renders.
-   */
-  id?: string;
```

Comment:
Actually, looks like this change [is not in main](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/notifications/toasts/toasts_api.tsx). I think it exists in the feature branch since it was cherry-picked, perhaps from 2.x?

Do you recall the owner of that change, and can we get that front ported into `main`?

Reply:
Oh shoot, yeah thanks catching that. Lemme look into why it's missing